rust/noosphere/src/platform.rs

```diff
@@ -19,13 +19,13 @@ mod inner {
     pub type PlatformKeyStorage = InsecureKeyStorage;
 
     #[cfg(not(feature = "ipfs-storage"))]
-    pub type PlatformStorage = NativeStorage;
+    pub type PlatformStorage = SledStorage;
```

Comment:
What is `platform` in this case and why is it different than whatever the alternative is?

Reply:
`PlatformStorage` here is the default storage used for a given platform for (mostly?) sphere storage, where its (given this patch) `IndexedDbStorage` (previously `WebStorage`) on wasm32 builds, and `SledStorage` (previously `NativeStorage`) elsewhere (though apple/aarch64 builds here are distinct, mostly paving the way for apple-specific key enclaves) -- this is where we'd compose different backends for different platforms e.g. maybe `SledStorage` for desktop builds but `RocksDbStorage` on iOS

Reply:
Though note that there's no functional changes in this patch, the sled backend is still used for non-web storage everywhere